src/components/site/SiteFooter.tsx

```diff
@@ -89,6 +89,16 @@ export const SiteFooter: React.FC<{
           </Script>
         </div>
       )}
+      {site?.metadata?.content?.ua && (
+        <div className="xlog-umami-analytics">
+          <Script
+            id="umami-analytics" strategy="afterInteractive"
+            async src="https://analytics.umami.is/script.js"
```

Comment:
Break line

Reply:
> Break line

```suggestion
            async
            src="https://analytics.umami.is/script.js"
```
Do you mean that?

Reply:
Fixed in https://github.com/Crossbell-Box/xLog/commit/59c688d437aae17dd1fb61529406b84ed2af6c3c .
Thanks #DIYgod !